pyfr/integrators/dual/pseudo/pseudosteppers.py

```diff
@@ -195,13 +195,6 @@ def __init__(self, *args, **kwargs):
 
         self._nstages = len(self.b)
 
-        # Register a kernel to multiply rhs with local pseudo time-step
-        self.backend.pointwise.register(
-            'pyfr.integrators.dual.pseudo.kernels.localdtau'
-        )
-
-        tplargs = dict(ndims=self.system.ndims, nvars=self.system.nvars)
-
         self.dtau_upts = proxylist([])
```

Comment:
Does this need to be a proxylist (also worth doing a once-over on our proxylist imports as some might not be needed).